bokehjs/src/coffee/models/ranges/data_range1d.coffee

```diff
@@ -22,7 +22,7 @@ export class DataRange1d extends DataRange
   }
 
   @internal {
-      mapper_hint:     [ p.String, 'auto' ]
+      scale_hint:     [ p.String, 'auto' ]
```

Comment:
(I only renamed it at all because it was easier to do and verify during the mass rename)